editor/src/core/workers/parser-printer/parser-printer-comments.spec.ts

```diff
@@ -209,18 +149,19 @@ describe('Parsing and printing code with comments', () => {
           // Comment inside inner JS
           return 2
         } // Comment after inner JS
+
         // Comment before return statement
         return (
           <div /* Comment before all attributes */
             data-uid='aaa' /* Comment inside attributes */
             someProp={/* Comment at start of JSX attribute */ 1000 /* Comment at end of JSX attribute */}
             someProp2={{
               /* Comment before object key */ someKey /* Comment after object key */:
-                /* Comment before object value */ 'someValue' /* Comment after object value */,
+                /* Comment before object value */ 'someValue' /* Comment after object separator */ /* Comment after object value */,
```

Comment:
The comments after an object or array separator is moved in front of the comment after the object or array value by prettier for some reason.

Reply:
oh I missed your comment when I wrote mine 1 minute ago :) 

eek! PR to prettier? ;)